core/app/models/spree/property.rb

```diff
@@ -6,7 +6,8 @@ class Property < Spree::Base
     has_many :product_properties, dependent: :delete_all, inverse_of: :property
     has_many :products, through: :product_properties
 
-    validates :name, :presentation, presence: true
+    validates :presentation, presence: true
+    validates :name, uniqueness: { scope: :presentation }, presence: true
```

Comment:
@priyank-gupta There might be a case when property name of the product is registered but it's presentation might be different